runtime/interpreter/value_test.go

```diff
@@ -2089,3 +2097,46 @@ func TestNumberValue_Equal(t *testing.T) {
 		})
 	}
 }
+
+func TestPublicKeyValue(t *testing.T) {
+
+	t.Parallel()
+
+	t.Run("Stringer output includes public key value", func(t *testing.T) {
+
+		t.Parallel()
+
+		publicKey := NewArrayValueUnownedNonCopying(
+			NewIntValueFromInt64(1),
+			NewIntValueFromInt64(7),
+			NewIntValueFromInt64(3),
+		)
+
+		publicKeyString := "[1, 7, 3]"
+
+		sigAlgo := func() *CompositeValue {
+			fields := NewStringValueOrderedMap()
+			fields.Set(sema.EnumRawValueFieldName, UInt8Value(sema.SignatureAlgorithmECDSA_secp256k1.RawValue()))
+
+			return &CompositeValue{
+				qualifiedIdentifier: sema.SignatureAlgorithmType.QualifiedIdentifier(),
+				kind:                sema.SignatureAlgorithmType.Kind,
+				fields:              fields,
+			}
```

Comment:
@SupunS is this construction of a non-deferred composite value here correct?

Reply:
This looks OK. We could also use the constructor, but that takes additional two params (owner and location).